app/main.py

```diff
@@ -1,8 +1,42 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int,
+
+    ) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_humans = []
+    for person in people:
+        wife = None
+        husband = None
+        if "wife" in person:
+            wife = person["wife"]
+        elif "husband" in person:
+            husband = person["husband"]
+        mate = Person(person["name"], person["age"])
+        mate.wife = wife
+        mate.husband = husband
```

Comment:
You are creating `wife` and `husband` variables but you are not using them properly. You are assigning them to the `person` dictionary but then you are not using these variables. Also, you are assigning `wife` and `husband` to `Person` instances as attributes, but these attributes are not defined in your `Person` class. This is not a good practice. It's better to define all attributes in the class definition. If a `Person` can have a wife or a husband, these should be attributes of the `Person` class. You should also check the type of `wife` and `husband`. They should be instances of the `Person` class, not strings.

Reply:
Please take into consideration this comment